src/pages/projects/CreateProject.tsx

```diff
@@ -111,7 +112,12 @@ const CreateProject: FC = () => {
       )
         .then(() => {
           navigate(`/ui/project/${values.name}/instances`);
-          toastNotify.success(`Project ${values.name} created.`);
+          toastNotify.success(
+            <>
+              Project <ResourceLabel bold type="project" value={values.name} />{" "}
```

Comment:
Should this be a link?

Reply:
I think it could be, do we link to the instance list for that project?

Reply:
perfect, I'm doing a project wide scan again to see if I can convert labels into links. So this will be part of that change :+1: